src/components/LogView/index.js

```diff
@@ -159,7 +187,7 @@ class LogView extends React.Component {
     if (findElements.length > 0) {
       let elem = findElements[0];
       let position = elem.getBoundingClientRect();
-      let windowHeight = window.innerHeight; // eslint-disable-line no-unused-vars
+      let _windowHeight = window.innerHeight;
```

Comment:
ah got it now. The underscore rule only applies to function arguments, not to variables. In this case, you should delete it